Client/Frontend/Browser/PrivacyHub/PrivacyHubAllTimeSection.swift

```diff
@@ -0,0 +1,132 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Shared
+import BraveShared
+import Data
+
+extension PrivacyReportsView {
+  struct PrivacyHubAllTimeSection: View {
+    @Environment(\.sizeCategory) private var sizeCategory
+    @Environment(\.horizontalSizeClass) private var horizontalSizeClass
+    
+    let allTimeMostFrequentTracker: CountableEntity?
+    let allTimeRiskiestWebsite: CountableEntity?
+    
+    let allTimeListTrackers: [PrivacyReportsTracker]
+    let allTimeListWebsites: [PrivacyReportsWebsite]
+    
+    private(set) var onDismiss: () -> Void
+    
+    private var allTimeTrackerView: some View {
+      VStack {
+        Text(Strings.PrivacyHub.allTimeTrackerTitle.uppercased())
+          .font(.caption)
+          .frame(maxWidth: .infinity, alignment: .leading)
+          .foregroundColor(Color(.secondaryBraveLabel))
+        
+        if let allTimeMostFrequentTracker = allTimeMostFrequentTracker {
+          VStack(alignment: .leading) {
+            Text(allTimeMostFrequentTracker.name)
+            
+            Text(String.localizedStringWithFormat(Strings.PrivacyHub.allTimeSitesCount,
+                                                  allTimeMostFrequentTracker.count))
+          }
+          .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .leading)
```

Comment:
Max height is to avoid height mismatch between two views placed next to each other,
for else case both views use the same text so it is not needed, I can add it for future proofing and safety though.
Here's example without maxHeight set

<img width="867" alt="Zrzut ekranu 2022-04-20 o 10 18 25" src="https://user-images.githubusercontent.com/6950387/164183533-14036f5b-e39a-47c5-a2bd-937255c49dd4.png">